meta-mender-commercial/recipes-mender/mender-gateway/mender-gateway_git.bb

```diff
@@ -0,0 +1,27 @@
+require mender-gateway.inc
+
+inherit mender-closed-source-tar-utils
+
+SRCREV = "${@mender_closed_source_srcrev_from_src_uri(d, '${SRC_URI}', 'mender-gateway')}"
+
+def mender_gateway_version_from_preferred_version(d, srcrev):
+    pref_version = d.getVar("PREFERRED_VERSION")
```

Comment:
Same here with regards to `PREFERRED_VERSION`.

Reply:
I fixed this one too and moved the function to the shared class, so that the fix gets into `mender-monitor` too.